src/lib/y2storage/md_level.rb

```diff
@@ -27,7 +27,25 @@ module Y2Storage
   # This is a wrapper for the Storage::MdLevel enum
   class MdLevel
     include StorageEnumWrapper
+    include Yast::I18n
 
     wrap_enum "MdLevel"
+
+    TRANSLATIONS = {
+      unknown: N_("Unknown"),
+      raid0:   N_("RAID0"),
+      raid1:   N_("RAID1"),
+      raid5:   N_("RAID5"),
+      raid6:   N_("RAID6"),
+      raid10:  N_("RAID10")
+    }
+
+    def to_human_string
+      textdomain "storage"
```

Comment:
well, I would like to have it simple. If we need translations on more places, we should move it to initialize